exonum-java-binding-core/src/test/java/com/exonum/binding/transaction/RawTransactionTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.transaction;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.exonum.binding.test.Bytes;
+import org.junit.jupiter.api.Test;
+
+class RawTransactionTest {
+
+  @Test
+  void builderTest() {
```

Comment:
Test for autogenerated builder is redundant as for me.

Reply:
it tests that autovalue stuff is properly configured